### PR TITLE
feat: simple script to fetch from flight api data given icao model nu…

### PR DESCRIPTION
**Pull request by AnthonyD4101**
…mber and distance

### CODE REVIEW

File: backend/flightData.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import requests
+from datetime import datetime
+
+def get_flight_data(aircraft: str, distance: int):
```
Comment (sami5436) [inline backend/flightData.py:4]: great. i appreciate how we can control both the aircraft and distance.

File: backend/flightData.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    url = f"https://despouy.ca/flight-fuel-api/q/?aircraft={aircraft}&distance={distance}"
```
Comment (sami5436) [inline backend/flightData.py:5]: did we double check this from the source????

File: backend/flightData.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    if response.status_code == 200:
+        flight_data = response.json()
+        return flight_data
+    
+    else:
+        print(f"Error: {response.status_code} - {response.text}")
+        return None
+    
+# api call testing
+aircraft = "60006b"
```
Comment (sami5436) [inline backend/flightData.py:24]: ahhh i was going to say hey whats this hard coding, but realized it was meant for testing.

File: backend/flightData.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+distance = 125
+
+flight_data = get_flight_data(aircraft, distance)
```
Comment (sami5436) [inline backend/flightData.py:27]: looks good overall. approval on my end

File: backend/flightData.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import requests
+from datetime import datetime
+
+def get_flight_data(aircraft: str, distance: int):
+    url = f"https://despouy.ca/flight-fuel-api/q/?aircraft={aircraft}&distance={distance}"
+
+    params = {
+        "icao": aircraft,
+        "distance": distance
+    }
```
Comment (Jynxae) [inline backend/flightData.py:10]: Great! Are we sure that these are the only params?

Comment (AnthonyD4101) [reply backend/flightData.py:10]: For now yes, we can also do batch queries, but I just want basic functionality for now

File: backend/flightData.py (hunk 4 above)
Comment (AnthonyD4101) [reply backend/flightData.py:27]: Thank you.

**Review by Jynxae — APPROVED** [review]
Other than that, great work Anthony! Just double check where you are getting this information and you should be good to go!

File: backend/flightData.py (hunk 2 above)
Comment (AnthonyD4101) [reply backend/flightData.py:5]: I tested against Postman request for the same parameters and got the same and expected response.

**Pull request merged**